### ISSUE TITLE
Another typos

### ISSUE THREAD

**Issue by Mates-K1** [ZenPhoto20/ZenPhoto20#795]
MySQL **versuib** 5.5.3 or **greter** is required to support trans-BMP character encod**ings..**

**Comment by Mates-K1** [comment]
![nepojmenovany 1](https://user-images.githubusercontent.com/7648991/41027029-57f6f934-6976-11e8-9bae-2a80531c00a9.jpg)

I think it is in the contact form.

**Comment by Mates-K1** [comment]
Sorry for out of topic - what does "templates" mean in Gallery stat on main backend page? 
I have there "3 templates". 
What kind of templates? Templates of what?

**Comment by sbillard** [comment]
The user groups provides two kinds of models for users. Simple groups 
get assigned to the user and his rights are derived from them. If the 
group rights are changed, the user automatically has his rights changes. 
Templates are a one time assignment of rights. The user gets the rights 
that the template had at the time of assignment. If the template rights 
subsequently change it has no effect on the user.

So, in short: groups are linked to the user, templates are initial value 
settings.


-Stephen

**Comment by Mates-K1** [comment]
Thank you for your explanation, I must admit that I did not know anything about the templates so far.
